memory-bank/techContext.md

```diff
@@ -35,6 +35,56 @@
 - **Testing**: Vitest for unit testing, Playwright for e2e testing.
   - **Supabase Testing Approach**: A direct testing approach is used with Supabase. We create real records in the database, run the actual functions with these records, and then clean up the test data afterwards. This provides more realistic tests that verify the actual functions with real database interactions, leveraging Supabase's ability to be executed directly in test environments.
 
+## Database Migration Workflow
+
+### Migration from Prisma to Supabase
+
+The project has completed the transition from Prisma ORM to Supabase for database migrations. While some parts of the application still use the Prisma client (hence `gen:prisma` script is retained), all database schema migrations are now handled through Supabase.
+
+### Supabase Migration Workflow
+
+The migration workflow follows Supabase's recommended practices:
+
+1. **Creating a new migration**:
+   ```bash
+   pnpm supabase:migration:new <migration_name>
```

Comment:
nits

( I don't use cline 🙏 but  )

cline can do this? I confirmed now: `$ pnpm --filter @liam-hq/db supabase:migration:new xxxx`  works, but `$ pnpm supabase:migration:new xxxx` results in error.

Reply:
Cline probably automatically run `cd frontend/packages/db &&`, but I'll update if it doesn't work.

Reply:
> `cd frontend/packages/db &&` 

I see. ok!